.github/workflows/build.yml

```diff
@@ -66,6 +66,18 @@ jobs:
       with:
         path: 'examples'
 
+  build-release-v4_4:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+      with:
+        submodules: 'recursive'
+    - name: esp-idf build
+      uses: espressif/esp-idf-ci-action@release-v4.4
```

Comment:
Suggest `uses: espressif/esp-idf-ci-action@latest` and later `with: esp_idf_version: release/v4.4`